tests/performance_tests/NodeosPluginArgs/generate_nodeos_plugin_args_class_files.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
```

Comment:
A file-level triple-quoted comment block describing the purpose and limitations of this script would be helpful.  It's easy enough to guess what it's for from the name I suppose, but a brief description of the output is probably beneficial.

Reply:
Addressed here: [792a0f8](https://github.com/AntelopeIO/leap/pull/546/commits/792a0f8559656403d63b8c87ee952ff6b4cee140)

Also added a suggestion around using the script to regenerate when validation test fails: [2e4495e](https://github.com/AntelopeIO/leap/pull/546/commits/2e4495e651a9df196a0dd5710c854b64aa585e68)